src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -221,13 +416,54 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         const auto backgroundImgCheckboxTooltip{ ToolTipService::GetToolTip(UseDesktopImageCheckBox()) };
         Automation::AutomationProperties::SetFullDescription(UseDesktopImageCheckBox(), unbox_value<hstring>(backgroundImgCheckboxTooltip));
 
+        const auto showAllFontsCheckboxTooltip{ ToolTipService::GetToolTip(ShowAllFontsCheckbox()) };
+        Automation::AutomationProperties::SetFullDescription(ShowAllFontsCheckbox(), unbox_value<hstring>(showAllFontsCheckboxTooltip));
+
         Automation::AutomationProperties::SetName(DeleteButton(), RS_(L"Profile_DeleteButton/Text"));
     }
 
+    IInspectable Profiles::CurrentFontFace() const
```

Comment:
This seems like it could be a member of ProfilesViewModel, and then we could hide the `CompleteFontList
` and `MonospaceFontList` from the PVM idl entirely

Reply:
Huh, this probably does make more sense in the PVM, actually. But we'll still need the two font lists exposed for the XAML, because we have 2 combo boxes (one for each list).